assets/solution/deployment.template.schema.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
will discuss posting this schema to the schema store instead of storing locally in the developer sources